langchain/chains/llm.py

```diff
@@ -184,13 +194,23 @@ async def aapply(
         await run_manager.on_chain_end({"outputs": outputs})
         return outputs
 
-    def create_outputs(self, response: LLMResult) -> List[Dict[str, str]]:
+    @property
+    def _run_output_key(self) -> str:
+        return self.output_key
+
+    def create_outputs(self, response: LLMResult) -> List[Dict[str, Any]]:
```

Comment:
I don't see a way to propagate llm_output to the end user because top level response is a list rather than dict and llm_output is across all inputs if I understand correctly.

But if there's a way to do it, we should do it, so users don't have to implement callbacks as a work-around to raw information missing.